### PR TITLE
Implement species filter and very basic (ugly) styling

### PR DESCRIPTION
**Pull request by j-andrews7**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  width: 100%;
+}
+
+/* Style query form */
+.queryform {
+    overflow: hidden;
+    background-color: #BCCDC7;
+    margin: auto;
+    padding: 10px;
+    margin: 20px 20px 0px 20px;;
```
Comment (melalawi) [inline css/style.css]: Extra semicolon.

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+.content {
+	background-color: #83A298;
+}
+
+/* Style the info */
+.info {
+    background-color: #BCCDC7;
+    padding: 10px;
+    margin: 0px 20px 0px 20px;
```
Comment (melalawi) [inline css/style.css]: Can be simplified to `margin: 0 20px;`. This is also seen elsewhere. Here's a helpful table.

| CSS Entry   | Description                                                                           |
|-------------------|--------------------------------------------------------------------------------------------------------------------------------|
| `margin: a;`       | Margin of size `a` will apply in all four directions                                                                           |
| `margin: a b;`     | Margin of size `a` will apply to top and bottom directions, margin of size `b` will apply to left and right directions         |
| `margin: a b c;`   | Margin of size `a` will apply to top, margin of size `b` will apply to left and right, margin of size `c` will apply to bottom |
| `margin: a b c d;` | Top margin of size `a`, right margin of size `b`, bottom margin of size `c`, left margin of size `d`                           |

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,78 @@
+* {
```
Comment (melalawi) [inline css/style.css]: Spacing issues across CSS file.

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+    text-align: center;
+    padding: 14px 16px;
+    text-decoration: none;
+}
+
+/* Change color on hover */
+.history a:hover {
+    background-color: #ddd;
+    color: black;
+}
```
Comment (melalawi) [inline css/style.css]: These `.history` rules don't seem to be in use.

File: js/query.js [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+        document.addEventListener("DOMContentLoaded", function(event) {
+            console.log("DOM fully loaded.");
+            document.getElementById("searchButton").addEventListener('click', newQuery);
+        });
+
+        function newQuery() {
+            var term = document.getElementById("query").value;
+            var query_url = "https://mygene.info/v3/query?q=" + term;
+            var species_options = document.getElementById("species");
+            if (species_options.selectedIndex !== -1) {
```
Comment (melalawi) [inline js/query.js]: 1. Add a space between variable declarations and business logic.
2. Use camelCasing in JavaScript.

File: js/query.js [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+                var species = species_options.options[species_options.selectedIndex].value;
+                var query_url = "https://mygene.info/v3/query?q=" + term + "&species=" + species;
```
Comment (melalawi) [inline js/query.js]: You needlessly redeclare the `query_url` variable. Remove `var`.

File: js/query.js [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+                                annotateGene(geneId);
+                            } else {
+                                document.getElementById("basics").textContent = 'No hits.'
+                                document.getElementById("summary").textContent = ''
+                            }
+                        });
+                    }
+                )
+                .catch(function(err) {
+                    console.log('Fetch Query Error :-S', err);
```
Comment (melalawi) [inline js/query.js]: Replace this `console.log()` call with `console.error()` for now. Also, get rid of that emoticon please.

File: js/query.js [hunk 8]
```diff
@@ -0,0 +1,69 @@
+(function() {
```
Comment (melalawi) [inline js/query.js]: General comment: Don't mix up quotes. Stick with single quotes throughout your JavaScript.

File: js/query.js [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+                        // Examine the text in the response
+                        response.json().then(function(data) {
+                            console.log(data);
+                            var summary = data.summary
+                            document.getElementById("summary").textContent = 'Gene Summary: ' + summary;
+                        });
+                    }
+                )
+                .catch(function(err) {
+                    console.log('Fetch Gene Annotation Error :-S', err);
```
Comment (melalawi) [inline js/query.js]: Above comment.

File: css/style.css (hunk 4 above)
Comment (j-andrews7) [reply css/style.css]: These are for a later feature - they will be utilized later.

**Comment by j-andrews7** [comment]
Requested changes made and noted for future reference. Up next is displaying all of the wanted data, dynamically creating links out to other databases, and modifying these fields for individual species.

**Pull request merged**